KarhooUISDK/UIComponents/LoyaltyView/KarhooLoyaltyViewMVP.swift

```diff
@@ -41,3 +48,25 @@ protocol LoyaltyPresenter {
 enum LoyaltyMode {
     case none, earn, burn
 }
+
+enum LoyaltyBalanceMode {
```

Comment:
Why can't it be nested inside the view component? There is no use of this type in global scope, I think.